python/src/aiconfig/editor/client/src/components/notifications/NotificationContext.tsx

```diff
@@ -0,0 +1,13 @@
+import { createContext } from "react";
+import { AIConfigEditorNotification } from "./NotificationProvider";
+
+/**
+ * Context for providing showNotification method throughout the editor
```

Comment:
Let's add a bit more detail in code. Ex: "For example, for VS Code extension we want to re-use VS Code's notification framework instead of using the one provided by Mantine"